packages/app-desktop/app.ts

```diff
@@ -492,7 +493,9 @@ class Application extends BaseApplication {
 		if (Setting.value('startMinimized') && Setting.value('showTrayIcon')) {
 			// Keep it hidden
 		} else {
-			bridge().window().show();
+			// bridge().window().show();
```

Comment:
Any reason you moved this call to ElectronAppWrapper?

Reply:
I took this path after your comment on the closed PR - https://github.com/laurent22/joplin/pull/7824#discussion_r1114846611. I will keep the call here and just hide the splash.